src/routes/api/article/article.routes.js

```diff
@@ -22,7 +23,7 @@ const {
 router.post('/:articleId/favorite', [auth, confirmEmailAuth, validateId], FavoritesController.createOrRemoveFavorite);
 router.post('/', [auth, confirmEmailAuth], imageUpload.array('images', 10), validate(schema.articleSchema), articleController.createArticles);
 router.get('/', checkQuery, articleController.getAllArticles);
-router.get('/:slug', articleController.getOneArticle);
+router.get('/:slug', StatsWare.saveStat, articleController.getOneArticle);
```

Comment:
Can an Unauthenticated user get his reading statistics?

Reply:
No Patric, it is the user stats, and the only way of identifying the user is through his auth token/userid or email extracted from here